README.md

```diff
@@ -28,13 +28,16 @@ Spin up as many Kong instances as you want. On different commits at the same
 time. With different OpenSSL, OpenResty and LuaRocks versions. Run a shell
 inside of the containers, make Kong roar. Make Kong fail, cd into the repo, fix
 it. Make Kong start again. Commit it. Push it, ship it!
+
 ## Synopsis
 
 ```
-Usage: gojira action [options...]
+                      Gojira (Godzilla)
+
+Usage: $GOJIRA action [options...]
 
 Options:
-  -t,  --tag            git tag to mount kong on (default: master)
+  -t,  --tag            git tag to mount kong on (default: $GOJIRA_TAG)
```

Comment:
We should also provide the default value for `GOJIRA_TAG` here. Maybe:

```
  -t,  --tag            git tag to mount kong on (default: $GOJIRA_TAG or "master")
```

We can update the `.sh` file to reflect this change.